src/opnsense/www/js/jquery.bootgrid.js

```diff
@@ -559,7 +559,7 @@ function renderRowCountSelection(actions)
         {
             var item = $(tpl.actionDropDownItem.resolve(getParams.call(that,
                 { text: getText(value), action: value })))
-                    ._bgSelectAria(value === that.rowCount)
+                    ._bgSelectAria(value.toString() === that.rowCount)
```

Comment:
less strict comparision (replacing `===` with `==` ) works for me too )

Reply:
@kulikov-a I also tried this first, but then I'll get a compiler warning:

```
Running "jshint:files" (jshint) task

   dist/jquery.bootgrid.js
    562 |                    ._bgSelectAria(value == that.rowCount)
                                                    ^ Expected '===' and instead saw '=='.

>> 1 error in 1 file
Warning: Task "jshint:files" failed. Use --force to continue.

Aborted due to warnings.
```

Therefore I've changed the left part to string to avoid the warning, as current grunt settings prevent the build with warnings.

Reply:
@8191 wow. sorry then) the difference between simple debug and building..